dockerfiles/base-ci-linux/Dockerfile

```diff
@@ -59,7 +59,10 @@ RUN set -eux; \
 # apt clean up
 	apt-get autoremove -y; \
 	apt-get clean; \
-	rm -rf /var/lib/apt/lists/*
+  rm -rf /var/lib/apt/lists/*; \
+# add non-root user
+  groupadd -g 1000 nonroot; \
+  useradd -u 1000 -g 1000 -s /bin/bash -m nonroot
```

Comment:
both `base-ci-linux` and `base-ci` are designed to be used as the base layers for the other images, other than that they are not used.
So AFAIU you do the preparation part here (for DRY sakes) and actual user's called from this boilerplate in the end images, am I right?

Reply:
Yes, that's right.